release/script_release.py

```diff
@@ -70,12 +70,13 @@ def __init__(self, release_type, remote_name="canalTP"):
         # TODO quit on error
         self.git.rebase(remote_name + "/dev", "dev")
         self.dev_data_version = self.get_data_version()
+        remote_release = remote_name + "/release"
         try:
-            self.git.checkout("release")
+            self.git.checkout("-B release " + remote_release)
```

Comment:
Minor, for consistency with line 79... Or the other way with '+' 
```suggestion
            self.git.checkout("-B release ", remote_release)
```